### PR TITLE
Sous 875

### PR DESCRIPTION
**Pull request by JC1738**
update deployment_builder.go to remove Vomitf and Debugf statements.

### CODE REVIEW

**Comment by sous-machine-account** [comment]
LGTM

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: ext/singularity/deployment_builder.go [hunk 1]
```diff
@@ -56,10 +57,10 @@ func (nsd nonSousError) Error() string {
 }
 
 func ignorableDeploy(log logging.LogSink, err error) bool {
```
Comment (samsalisbury) [inline ext/singularity/deployment_builder.go]: Should this be `ignorableError` or is it talking about a deploy somehow?

File: ext/singularity/deployment_builder.go [hunk 2]
```diff
@@ -56,10 +57,10 @@ func (nsd nonSousError) Error() string {
...
-	log.Vomitf("checking to see if %+v is ignorable", err)
+	messages.ReportLogFieldsMessage("checking to see if error is ignorable", logging.ExtraDebug1Level, log, err)
 	switch errors.Cause(err).(type) {
 	case nonSousError, notThisClusterError:
-		log.Vomitf("ignorable: %v", err)
+		messages.ReportLogFieldsMessage("ignorable", logging.ExtraDebug1Level, log, err)
```
Comment (samsalisbury) [inline ext/singularity/deployment_builder.go]: Maybe `"error not related to this cluster or sous"` to be specific. Although "not related to sous" seems an odd idea.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ with respect to its command line interface and HTTP interface.
 
 ## [Unreleased](//github.com/opentable/sous/compare/0.5.65...master)
 
+* All: Update logging package: deployment_builder.go
```
Comment (samsalisbury) [inline CHANGELOG.md]: Maybe "All: deployment builder now emits structured logs." ? Just to be specific about the externally visible change.

Comment (JC1738) [reply CHANGELOG.md]: @samsalisbury Made the changelog change, and pushed to the PR

**Comment by JC1738** [comment]
@samsalisbury Yes, agree with your comments.  Wasn't sure of the original context of how these logs helped in first place, they are Vomit, so they should be not even emitted unless log level is high enough.  I would imagine next person who needs to do logic changes in here would update the logs to fit their need.  I take vomit and debug useful for the developer who is getting things working versus actual useful logging information for a larger audience, it's pretty personalized, where is Warning/Information should be logs that consumed by everyone.  Thoughts?

**Pull request merged**